src/mapper/github.mapping.ts

```diff
@@ -0,0 +1,64 @@
+import { getReposList, getRepoInfo } from '../client-services/github.client-service'
+
+interface RepoInfo {
```

Comment:
What is the intention of this interface?

After we understand the intention I think we should move this to another place.

Reply:
I have JSON-type data, including name, description, URL, etc. In the first loop, only the name of each repository is pushed to the object. When I tried without interface, it returned an error. 

> Argument of type '{ name: any; }' is not assignable to parameter of type '{ name: string; description?: string; url: string; languages: string[]; language: string; contributors: { name: string; avatar_url: string; }[]; }'.
>   Type '{ name: any; }' is missing the following properties from type '{ name: string; description?: string; url: string; languages: string[]; language: string; contributors: { name: string; avatar_url: string; }[]; }': url, languages, language, contributors

Reply:
Understand,

There are two suggestions.

1) Keep file clean, we should only have 1 class or interface per file. In the future we will turn on linter rules to prevent this.
2) I think your error is related to strict type rules in TS. In my PR, the rules will be relaxed a little bit.

Don't worry about this comment for now. We can focus on merging the linter rules into this PR and changes to #7.